utils/file_utils.jsx

```diff
@@ -35,3 +35,31 @@ export function trimFilename(filename) {
 
     return trimmedFilename;
 }
+
+export function getFileDimentionsForDisplay(dimentions, {maxHeight, maxWidth}) {
```

Comment:
about all my previous comments of making the second arg a constant, does it actually need to be an object? why not just passing `maxHeight` and `maxWidth` as parameters?

Reply:
Yes we could pass them as params.

Why object vs param:

1. They are naturally grouped into two categories making a better choice to have objects.
2. Better readability from the code we are calling this as it might not be obvious what the params are and the order in which we pass.

 ```
getFileDimentionsForDisplay(this.props.images[attachment.image_url], {maxHeight: 300, maxWidth: 500});

getFileDimentionsForDisplay(this.props.images[attachment.image_url], 300, 500});(We could define the values as constants to help but never the less the above code seems more readable to me)
```

3. Possible optional/default value later. If we have to use a default value for maxWidth or maxHeight though we might add a es6 default we would end up passing `null` to function invocations to keep the order of params

I see that we use params mostly in code and lot of places where we end up passing 4,5 params with some of them `false`, `null` as value making it really hard to read. And adding one more param to those functions makes us going through all instances of adding a null or false to the last optional to keep the order.

My thumb rule is to have objects 

1. If we have any logical grouping in arguments 
2. More than 3 arguments.
3. if we have optionals(At least optional should be part of object than spread out as params)

Are there any advantages of passing them as params?

> If there are hardly added advantages of using params to objects. Why use them as they do pose more downsides?